sway-lib-std/src/u128.sw

```diff
@@ -32,6 +32,19 @@ impl From<(u64, u64)> for U128 {
     }
 }
 
+impl From<u64> for U128 {
+    fn from(num: u64) -> U128 {
+        U128 {
+            upper: 0,
+            lower: num,
+        }
+    }
+
+    fn into(self) -> u64 {
+        self.as_u64().unwrap()
+    }
```

Comment:
Here, `into` may fail, but the Rust semantics says that the `From`/`Into` trait methods must not fail (see https://doc.rust-lang.org/std/convert/trait.From.html).